### DISCUSSION TITLE
Why does code server ask for login again so fast when I just switch to another app for a while?

### DISCUSSION THREAD

**Discussion by unclecode** [coder/code-server#5784] [Q&A]
Say I turn on the live server while I am in code-server, then switch to Safari to check the result, then if it takes about 40-60 seconds when I return to the code server it asks me for the login password and then the live server isn't there and everything starts over again, so I have to kill the process on the port the live server was listening on and start it again. Is there a way to increase or disable it?

**Comment by jsjoeio** [comment]
Hmm...could you provide steps to reproduce this issue? Happy to dig in!
